src/encoded/static/scss/encoded/modules/_forms.scss

```diff
@@ -63,21 +63,27 @@ input[type="text"], input[type="password"], input[type="date"], input[type="date
 
 // Navbar search
 #navbar-search {
+	background-color: white;
 	width: 180px;
 	border-radius: 14px;
-	height: 27px;
+	height: 25px;
 	padding: 3px 26px 4px 14px;
 	padding-top: 3px;
 	padding-bottom: 4px;
 	margin-top: 3px;
+	border: none;
+	box-shadow: none !important;
+	font-family: inherit;
+	font-size: 1rem;
+	font-weight: normal;
```

Comment:
It's ok to leave these, but I think lines 76-78 aren't necessary. `#navbar-search` will have `font-size: 1rem` and `font-weight: normal` already, from parent elements and `<input>` elements are already set to have `font-family: inherit`. If it seems clearer to have these explicitly set, though, that's also good.

Reply:
Good point. Corrected. I refactored the code from somewhere else and failed to realize these were not needed.